src/public/js/management.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function updateNewShiftModalValues(username: string) {
+  (document.getElementById('newShiftUsername')! as HTMLInputElement).value =
```

Comment:
Check if newShiftUsername exists before assigning a value.
```suggestion
  const newShiftUsername = document.getElementById('newShiftUsername') as HTMLInputElement
  if (newShiftUsername) {
    newShiftUsername.value = username;
  }
```